.env.example

```
@@ -1 +1,17 @@
 BLOG_API_KEY=your-api-key-here
+
+# Step 9: admin workspace auth
+ADMIN_USERNAME=admin
+ADMIN_PASSWORD_HASH=$argon2id$...
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ADMIN_PASSWORD_HASH key should go before the ADMIN_USERNAME key

(UnorderedKey)